docs/src/development/ssh/troubleshoot-ssh.md

```diff
@@ -66,15 +66,48 @@ Make sure you have added your public SSH key to your provider and that your user
 
 ## Add a second authentication factor
```

Comment:
```suggestion
## Using a second authentication factor
```

Reply:
Well, one of the paths is about adding it to your account if you don't have it already. I'm not sure, so I'll just leave it as is.